plugin/inlay_hint.py

```diff
@@ -21,7 +21,9 @@ class LspToggleInlayHintsCommand(LspWindowCommand):
 
     def __init__(self, window: sublime.Window) -> None:
         super().__init__(window)
-        window.settings().set('lsp_show_inlay_hints', userprefs().show_inlay_hints)
+        settings = userprefs()
+        default = settings.show_inlay_hints if settings is not None else False
+        window.settings().set('lsp_show_inlay_hints', default)
```

Comment:
Might not be a new thing but shouldn't it use the already saved setting from window, if set? It seems like it's potentially overriding what is stored there already.

Reply:
But this is the first place where it gets saved in the window settings, so I think it can't be already set. The only other place where it would be saved in the window settings would be from the `run` method.

Reply:
When starting ST the windows are restored including the window settings so it could be stored already.